command/repo.go

```diff
@@ -189,7 +189,7 @@ func repoCreate(cmd *cobra.Command, args []string) error {
 	if len(args) > 0 {
 		name = args[0]
 		if strings.Contains(name, "/") {
-			newRepo := ghrepo.FromFullName(name)
+			newRepo, _ := ghrepo.FromFullName(name)
```

Comment:
If it's possible to return an error in the current context, let's actually handle errors instead of ignoring them. Only assign the error value to `_` if it's not possible to return an error (i.e. if the current context does not have an error return value)

Reply:
Totally, done (a192e85a3fe61688d6b09a2614f54255f12a32e0)...feel free to confirm makes sense in the context of `repoCreate`. I believe so but still getting familiar with the codebase.